src/routing_table/mod.rs

```diff
@@ -806,24 +801,6 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
             .collect()
     }
 
-    fn initialise_merging_own_group(&self,
-                                    mut merge_details: OwnMergeDetails<T>)
-                                    -> OwnMergeState<T> {
-        merge_details.sender_prefix = self.our_group_prefix;
-        merge_details.groups
-            .extend(self.groups.iter().filter_map(|(prefix, names)| {
-                if names.is_empty() {
-                    None
-                } else {
-                    Some((*prefix, names.clone()))
-                }
-            }));
-        let mut our_group = self.our_group.clone();
-        our_group.insert(self.our_name);
-        let _ = merge_details.groups.insert(self.our_group_prefix, our_group);
-        OwnMergeState::Initialised { merge_details: merge_details }
-    }
-
```

Comment:
Looks like we don't use `OwnMergeState::Initialised` any more after this change - maybe it should be removed, then.